# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmoe-wayland/hotfix-gcc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cosmoe-wayland/headers/os/BeBuild.h.vanilla	2025-06-28 19:46:18.894930311 +0200
+++ cosmoe-wayland/headers/os/BeBuild.h	2025-06-28 19:46:38.624254332 +0200
@@ -55,7 +55,7 @@
 
 #if __GNUC__ == 2
 #	define B_HAIKU_ABI					B_HAIKU_ABI_GCC_2_HAIKU
-#elif (__GNUC__ >= 4 && __GNUC__ <= 14) || defined(__TINYC__)
+#elif (__GNUC__ >= 4 && __GNUC__ <= 15) || defined(__TINYC__)
 #	define B_HAIKU_ABI					B_HAIKU_ABI_GCC_4
 #else
 #	error Unsupported compiler!
